### ISSUE TITLE
zellij start is slow on mac big sur chip m1

### ISSUE THREAD

**Issue by makarono** [zellij-org/zellij#784]
Zellij versions > 0.15 are really slow when starting on Mac OS Big Sur. It lasts 4.5 second to start a fresh instance, or to attach to existing sessions. I tried on Linux debian and arch and that is not the case, it simply works flawlessly.
This is a far better terminal multiplexer from others and I was loosing already 5 days to discover what is the problem, the only thing that I was able to discover is when I dump config into a file and comment all plugins in config file the program starts fast as it should. Thank you and keep the good work. 
Br,
David

**Comment by imsnif** [comment]
Sorry for the experience, @makarono - and thanks for investigating.

@TheLostLambda - do you have any guess as to what might be causing this?

**Comment by makarono** [comment]
@imsnif just to add i tried all versions > 15.0 and < 0.18.1 and all have the same issue. Disabling plugins I tried only in 0.18.1 version. If you need any help with testing I will gladly help

**Comment by makarono** [comment]
Hi @TheLostLambda , thank you for response 

1. yes , when I disable all plugins everything  works full speed. If I disable one or two plugins it is still slow. I tried disabling default plugins in all possible combinations but it works snappy only when I disable all.
2. when plugins are active opening new tab also lasts 4,5 sec.
3. Yes I tried to build all versions > 0.15 but unsuccessfully due to mac os libc version.  This was the error which I get when try building:

cargo build --release
    Updating crates.io index
error: failed to select a version for the requirement `libc = "=0.2.98"`
candidate versions found which didn't match: 0.2.97, 0.2.96, 0.2.95, ...
location searched: crates.io index
required by package `zellij-utils v0.19.0 (/private/tmp/zellij/zellij-utils)`
